avd_docs/dockerfile/general/AVD-DS-0029/docs.md

```diff
@@ -1,14 +1,13 @@
 
-Do not install packages because they may be needed, install them only if you require them.
+'apt-get' install should use '--no-install-recommends' to minimize image size.
 
 ### Impact
-
-Image will increase substantially in size without `--no-install-recommends` for `apt-get install`.
+<!-- Add Impact here -->
 
 <!-- DO NOT CHANGE -->
 {{ remediationActions }}
 
 ### Links
 - https://docs.docker.com/develop/develop-images/dockerfile_best-practices/
-- https://ubuntu.com/blog/we-reduced-our-docker-images-by-60-with-no-install-recommends
+
```

Comment:
is this change related?

Reply:
this change occur when I ran make docs command before pushing code. This change coming in my all PR's. It is created automatically. I added it in the commit.

Reply:
this change occur when I ran make docs command before pushing code. This change coming in my all PR's. It is created automatically. I added it in the commit.